server/src/test/java/org/elasticsearch/index/analysis/PreConfiguredTokenFilterTests.java

```diff
@@ -132,8 +132,8 @@ public boolean incrementToken() {
         );
         assertSame(tff_v1_1, tff_v1_2);
 
-        byte major = VersionUtils.getFirstVersion().major;
-        Version version2 = Version.fromString(major - 1 + ".0.0");
+        int beforeAll = IndexVersionUtils.getFirstVersion().id() - 100;
+        IndexVersion version2 = IndexVersion.fromId((beforeAll / 1_000_000) * 1_000_000);
```

Comment:
This should be enough to capture 'version definitely before the first version', as we bump by 1,000,000 every major release

Reply:
Good idea - spotted an existing method I could use